tests/test_belief_manager.py

```diff
@@ -385,9 +385,9 @@ def check_position(self, expected):
     Returns:
       True if the check passes, False otherwise. """
     position = self._filter.position()
-    if abs(position[self._X] - expected[self._X]) > 0.5:
+    if abs(position[self._X] - expected[self._X]) > 1:
```

Comment:
Interesting choice, since this is a unit test, and the input data appears as-yet unchanged.